app/forms/questionnaire_form.py

```diff
@@ -10,16 +10,25 @@
 
 from app.forms.field_handlers import DateHandler, get_field_handler
 from app.forms.validators import DateRangeCheck, MutuallyExclusiveCheck, SumCheck
+from app.questionnaire.value_source_resolver import ValueSourceResolver
 
 logger = logging.getLogger(__name__)
 
 
 class QuestionnaireForm(FlaskForm):
     def __init__(
-        self, schema, question_schema, answer_store, metadata, location, **kwargs
+        self,
+        schema,
+        question_schema,
+        answer_store,
+        list_store,
+        metadata,
+        location,
+        **kwargs,
     ):
         self.schema = schema
         self.question = question_schema
+        self.list_store = list_store
         self.answer_store = answer_store
```

Comment:
The ordering here should match the init param order
```suggestion
        self.answer_store = answer_store
        self.list_store = list_store
```